assets/js/notice.js

```diff
@@ -0,0 +1,60 @@
+( function() {
+	wp.domReady( function() {
```

Comment:
Thanks @florianbrinkmann for the PR. I've just checked out the code and tried to test it. In my local development environment I just get a `TypeError: wp.domReady is not a function` error notice on the console.

Is this a problem of my setup or are any depencencies (a WP Editor JavaScript not loaded yet etc.).

Reply:
@patrickrobrecht oh, yes, there was a dependency missing. Just tried to reproduce the error and disabled all plugins, but did not get it. Something on my local install seems to load that already 🤔 Added the dependency that should be correct and changed the `wp-core-data` dependency to `wp-data`, I guess/hope that is the right one.

Reply:
Thanks for your fast reply and the fix. In my local WordPress 5.4 it's still not working due to the variable `postId` being `null`, although executing `wp.data.select('core/editor').getCurrentPostId();` in the console leads to the correct result. Maybe there's a problem with the order of scripts are loaded?

Reply:
Hm, does that happen in all cases (adding a new post and editing an existing one)?

Reply:
Yes, the error occurred in both cases. Moving the line `postId = ...` into the function of the subscribe block fixes the issue for me (I pushed a bugfix commit to your branch). Can you verify that?

Reply:
Yes, works for me, too. Should we maybe move the declaration of the `postId` variable into the `if ( isSaving && ! isChecking ) {` to not fire it that often? I am not sure if it has a negative impact on the editor performance to get it with every change, was just an idea.